articles/quickstart/spa/react/01-login.md

```diff
@@ -126,25 +126,32 @@ The Auth0 React SDK helps you retrieve the [profile information](https://auth0.c
 import React from "react";
 import { useAuth0 } from "@auth0/auth0-react";
 
-const Profile = () => {
+const UserProfile = () => {
   const { user, isAuthenticated, isLoading } = useAuth0();
 
   if (isLoading) {
     return <div>Loading ...</div>;
   }
 
-  return (
-    isAuthenticated && (
+  if (isAuthenticated) {
```

Comment:
Personally I am more a fan of showing:

```
return (
  isAuthenticated ? (
    <div>
      <img src={user.picture} alt={user.name} />
      <h2>{user.name}</h2>
      <p>{user.email}</p>
    </div>
  ) : ''
);
```

What do you think about this @stevehobbsdev and @Jay-study-nildana ? This avoids returning false without adding additional markup.

Reply:
@frederikprijck @stevehobbsdev 

See, there you go. I cannot do these nifty little pieces of code yet :P I like this one better.  It looks better too.